reference/Upload-API.yaml

```diff
@@ -319,65 +330,89 @@ paths:
                         type: array
                         uniqueItems: true
                         minItems: 1
-                        description: CloudMatrix attribute where value is a collection
+                        description: CloudMatrix attribute where value is a collection.
                         items:
                           type: object
                           additionalProperties: false
                           properties:
                             value:
                               type: string
                               minLength: 1
-                              description: CloudMatrix attribute value
+                              description: CloudMatrix attribute value (media asset configured custom field value)
                           required:
                             - value
                     required:
                       - id
               required:
                 - attributes
             examples:
-              Example:
+              String Value Example:
                 value:
                   attributes:
                     - id: e078d72a-aad4-4b68-9227-e0bde0b35c6d
-                      label: title
-                      value: Some Title
+                      value: 'Updated Title'
+              Integer Value Example:
+                value:
+                  attributes:
                     - id: 57e816fb-03b7-4706-a99f-5c6ef8d18ada
-                      label: description
-                      value: Some Description
+                      value: '123'
+              Boolean Value Example:
+                value:
+                  attributes:
                     - id: 57e816fb-03b7-4706-a99f-5c6ef8d18adb
-                      label: custom field
-                      value: Some custom field
+                      value: 'true'
+              Multi-Value Example:
+                value:
+                  attributes:
                     - id: e078d72a-aad4-4b68-9227-e0bde0b35c6z
-                      label: entitlement
                       items:
-                        - value: entitlement1
-                        - value: entitlement2
-                    - id: a61de76c-e367-4c63-93e0-2d6fd6e256cf
-                      label: Teams
-                      items:
-                        - value: Team A
-                        - value: Team B
-        description: The body should contain the additional CloudMatrix Metadata attributes you would like to update.
+                        - value: 'team1'
+                        - value: 'team2'
+              Date Value Example:
+                value:
+                  attributes:
+                    - id: 7ac5484e-ec8c-45f6-9437-95c909f5d41d
+                      value: '2022-02-24'
+              Time Value Example:
+                value:
+                  attributes:
+                    - id: 17cfb9e1-1775-41a8-8628-1cf053551c73
+                      value: '15:30'
```

Comment:
I tried with 7:14 for example but understand that format could be HH:mm :)

Reply:
Yep not sure 7:14 is ISO 8061 so yes must be HH:mm